spec/spec_helper.rb

```diff
@@ -16,7 +16,22 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.